src/ucp/core/ucp_listener.c

```diff
@@ -232,16 +232,16 @@ ucp_listener_conn_request_cb(uct_listener_h listener, void *arg,
                                   remote_data->conn_priv_data_length,
                                   "ucp_conn_request_h");
     if (ucp_conn_request == NULL) {
-        ucs_error("failed to allocate connect request, rejecting connection request %p on TL listener %p, reason %s",
-                  conn_request, listener, ucs_status_string(UCS_ERR_NO_MEMORY));
+        ucs_error("failed to allocate connect request, rejecting connection request %p on TL listener %p",
```

Comment:
line too long

Reply:
but we usually don't split strings following to linux-kernel code style